pwa/src/components/common/header.tsx

```diff
@@ -1,25 +1,25 @@
 import * as React from "react";
+import {HeaderContext} from "../../context/headerContext";
 
+const Header = () => {
+  const [header, _] = React.useContext(HeaderContext)
```

Comment:
Because the unused part of this `state` (`setHeader`, which you're saving as `_` right now) is the second value in the array, you can just skip it altogether. Like so: 

`const [header] = React.useContext(HeaderContext)`.